src/eventProcessor/GoogleAnalyticsEventProcessor.js

```diff
@@ -115,6 +115,27 @@ class GoogleAnalyticsEventProcessor {
     return 'googleAnalytics';
   }
 
+  /**
+   * Builds the POST request URL using `measurement_id` and `api_secret`
+   * query parameters if available
+   * @return {string}
+   * @private
+   */
+  buildRequestUrl() {
+    const measurementIdQuery =
+        this.measurementId_ ? `measurement_id=${this.measurementId_}` : '';
```

Comment:
Since these are arbitrary user inputs, what do you think about passing them through encodeURIComponent?